backend/src/ee/services/group/group-fns.ts

```diff
@@ -285,6 +287,63 @@ export const addUsersToGroupByUserIds = async ({
   });
 };
 
+/**
+ * Add identities with identity ids [identityIds] to group [group].
+ * @param {group} group - group to add identity(s) to
+ * @param {string[]} identityIds - id(s) of organization scoped identity(s) to add to group
+ */
+export const addIdentitiesToGroup = async ({
+  group,
+  identityIds,
+  identityDAL,
+  identityOrgMembershipDAL,
+  identityGroupMembershipDAL
+}: TAddIdentitiesToGroup) => {
+  return identityDAL.transaction(async (tx) => {
+    const identityIdsSet = new Set(identityIds);
+    const identityIdsArray = Array.from(identityIdsSet);
+
+    const foundIdentities = await identityOrgMembershipDAL.findByIds(identityIdsArray, group.orgId, tx);
```

Comment:
Use `membershipDAL` and not `identityOrgMembershipDAL`. Missing this in my last review. My bad